tests/helpers/worker.ts

```diff
@@ -2,14 +2,12 @@ import Piscina from 'piscina'
 import { makePiscina } from '../../src/worker/piscina'
 import { defaultConfig } from '../../src/config'
 import { LogLevel } from '../../src/types'
-import { mockJestWithIndex } from './plugins'
 
-export function setupPiscina(workers: number, code: string, tasksPerWorker: number): Piscina {
+export async function setupPiscina(workers: number, tasksPerWorker: number): Promise<Piscina> {
```

Comment:
Why `async` if it doesn't `await`?

Reply:
it did await at some point, but I removed it since and forgot to update the function. thanks for catching!